Sources/xcproj/CommentedString.swift

```diff
@@ -32,13 +32,21 @@ struct CommentedString {
 
         var escaped = string
         // escape escape
-        escaped = escaped.replacingOccurrences(of: "\\", with: "\\\\")
+        if escaped.contains("\\" as Character) {
```

Comment:
Isn't this part of another PR?

Reply:
Yup. I made the change in master so when targeting 2.0.0 it wants to merge everything. I have that fixed and will push soon.